### ISSUE TITLE
codelyzer was not updated by `npm update`

### ISSUE THREAD

**Issue by BryanWilhite** [BryanWilhite/angular.io-index-app#15]
`npm outdated`:

```console
Package                                  Current         Wanted         Latest  Location
@angular/animations                       5.2.11         5.2.11          6.1.2  index-app
@angular/cdk                               5.2.5          5.2.5          6.4.3  index-app
@angular/cli                               1.7.4          1.7.4          6.1.3  index-app
@angular/common                           5.2.11         5.2.11          6.1.2  index-app
@angular/compiler                         5.2.11         5.2.11          6.1.2  index-app
@angular/compiler-cli                     5.2.11         5.2.11          6.1.2  index-app
@angular/core                             5.2.11         5.2.11          6.1.2  index-app
@angular/flex-layout               5.0.0-beta.14  5.0.0-beta.14  6.0.0-beta.17  index-app
@angular/forms                            5.2.11         5.2.11          6.1.2  index-app
@angular/http                             5.2.11         5.2.11          6.1.2  index-app
@angular/language-service                 5.2.11         5.2.11          6.1.2  index-app
@angular/material                          5.2.5          5.2.5          6.4.3  index-app
@angular/platform-browser                 5.2.11         5.2.11          6.1.2  index-app
@angular/platform-browser-dynamic         5.2.11         5.2.11          6.1.2  index-app
@angular/router                           5.2.11         5.2.11          6.1.2  index-app
@types/node                              8.10.24        8.10.24         10.5.7  index-app
codelyzer                                  4.4.2          4.4.3          4.4.3  index-app
html-webpack-plugin                       2.30.1         2.30.1          3.2.0  index-app
jasmine-core                               2.8.0          2.8.0          3.2.0  index-app
karma                                      2.0.5          2.0.5          3.0.0  index-app
karma-coverage-istanbul-reporter           1.4.3          1.4.3          2.0.1  index-app
karma-jasmine-html-reporter                0.2.2          0.2.2          1.2.0  index-app
rxjs                                      5.5.11         5.5.11          6.2.2  index-app
ts-node                                    3.3.0          3.3.0          7.0.0  index-app
tslint                                     5.9.1          5.9.1         5.11.0  index-app
typescript                                 2.5.3          2.5.3          3.0.1  index-app
webpack                                   3.12.0         3.12.0         4.16.5  index-app
webpack-dev-server                        2.11.2         2.11.2          3.1.5  index-app
```

**Comment by BryanWilhite** [comment]
`npm audit`:

```console
                       === npm audit security report ===

# Run  npm install karma@3.0.0  to resolve 6 vulnerabilities
SEMVER WARNING: Recommended action is a potentially breaking change

  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > cryptiles > boom
                  > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Memory Exposure

  Package         tunnel-agent

  Dependency of   karma [dev]

  Path            karma > log4js > loggly > request > tunnel-agent

  More info       https://nodesecurity.io/advisories/598




  Low             Regular Expression Denial of Service

  Package         timespan

  Dependency of   karma [dev]

  Path            karma > log4js > loggly > timespan

  More info       https://nodesecurity.io/advisories/533



# Run  npm update original --depth 4  to resolve 1 vulnerability

  High            Open Redirect

  Package         url-parse

  Dependency of   webpack-dev-server [dev]

  Path            webpack-dev-server > sockjs-client > eventsource > original
                  > url-parse

  More info       https://nodesecurity.io/advisories/678



# Run  npm update url-parse --depth 3  to resolve 1 vulnerability

  High            Open Redirect

  Package         url-parse

  Dependency of   webpack-dev-server [dev]

  Path            webpack-dev-server > sockjs-client > url-parse

  More info       https://nodesecurity.io/advisories/678




                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > boom >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > cryptiles
                  > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > sntp >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > cryptiles >
                  boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566

found 28 vulnerabilities (1 low, 25 moderate, 2 high) in 17592 scanned packages
  run `npm audit fix` to fix 2 of them.
  6 vulnerabilities require semver-major dependency updates.
  20 vulnerabilities require manual review. See the full report for details.
```

**Comment by BryanWilhite** [comment]
ran `npm --save-dev i codelyzer@4.4.3` and `npm audit fix`:

```console:
                       === npm audit security report ===

# Run  npm install karma@3.0.0  to resolve 6 vulnerabilities
SEMVER WARNING: Recommended action is a potentially breaking change

  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > cryptiles > boom
                  > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Prototype pollution

  Package         hoek

  Dependency of   karma

  Path            karma > log4js > loggly > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566




  Moderate        Memory Exposure

  Package         tunnel-agent

  Dependency of   karma [dev]

  Path            karma > log4js > loggly > request > tunnel-agent

  More info       https://nodesecurity.io/advisories/598




  Low             Regular Expression Denial of Service

  Package         timespan

  Dependency of   karma [dev]

  Path            karma > log4js > loggly > timespan

  More info       https://nodesecurity.io/advisories/533




                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > boom >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > cryptiles
                  > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > sntp >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > cryptiles >
                  boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566

found 26 vulnerabilities (1 low, 25 moderate) in 17592 scanned packages
  6 vulnerabilities require semver-major dependency updates.
  20 vulnerabilities require manual review. See the full report for details.
```

**Comment by BryanWilhite** [comment]
ran `npm install karma@3.0.0` (which might cause breaking changes):

```console
                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > libcipm > npm-lifecycle > node-gyp > request > hawk >
                  sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > node-gyp > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > boom >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > cryptiles
                  > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-lifecycle > node-gyp > request > hawk > sntp >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > cryptiles >
                  boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > npm-registry-client > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > cryptiles > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   npm

  Path            npm > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566

found 20 moderate severity vulnerabilities in 17180 scanned packages
  20 vulnerabilities require manual review. See the full report for details.
```

```console
Package                                  Current         Wanted         Latest  Location
@angular/animations                       5.2.11         5.2.11          6.1.2  index-app
@angular/cdk                               5.2.5          5.2.5          6.4.3  index-app
@angular/cli                               1.7.4          1.7.4          6.1.3  index-app
@angular/common                           5.2.11         5.2.11          6.1.2  index-app
@angular/compiler                         5.2.11         5.2.11          6.1.2  index-app
@angular/compiler-cli                     5.2.11         5.2.11          6.1.2  index-app
@angular/core                             5.2.11         5.2.11          6.1.2  index-app
@angular/flex-layout               5.0.0-beta.14  5.0.0-beta.14  6.0.0-beta.17  index-app
@angular/forms                            5.2.11         5.2.11          6.1.2  index-app
@angular/http                             5.2.11         5.2.11          6.1.2  index-app
@angular/language-service                 5.2.11         5.2.11          6.1.2  index-app
@angular/material                          5.2.5          5.2.5          6.4.3  index-app
@angular/platform-browser                 5.2.11         5.2.11          6.1.2  index-app
@angular/platform-browser-dynamic         5.2.11         5.2.11          6.1.2  index-app
@angular/router                           5.2.11         5.2.11          6.1.2  index-app
@types/node                              8.10.24        8.10.24         10.5.7  index-app
html-webpack-plugin                       2.30.1         2.30.1          3.2.0  index-app
jasmine-core                               2.8.0          2.8.0          3.2.0  index-app
karma-coverage-istanbul-reporter           1.4.3          1.4.3          2.0.1  index-app
karma-jasmine-html-reporter                0.2.2          0.2.2          1.2.0  index-app
rxjs                                      5.5.11         5.5.11          6.2.2  index-app
ts-node                                    3.3.0          3.3.0          7.0.0  index-app
tslint                                     5.9.1          5.9.1         5.11.0  index-app
typescript                                 2.5.3          2.5.3          3.0.1  index-app
webpack                                   3.12.0         3.12.0         4.16.5  index-app
webpack-dev-server                        2.11.2         2.11.2          3.1.5  index-app
```

**Comment by BryanWilhite** [comment]
updating karma did not break existing tests:

![image](https://user-images.githubusercontent.com/5226627/43982854-aa2637c4-9cac-11e8-9d70-fc6f3c016ba0.png)
